docs/content/python/guides/roles/django.md

```diff
@@ -8,9 +8,9 @@ For more on using Oso with Django, see:
 - [django-oso reference](/reference/frameworks/django)
 - [Django data filtering tutorial](/guides/data_access/django)
 
-{{% callout "Data filtering and roles" "blue" %}}
+{{% callout "Data filtering and Oso roles" "blue" %}}
 
-The builtin roles feature is not currently supported
+The Oso roles feature is not currently supported
```

Comment:
Follow-up: standardize on capitalization of "Oso Roles" or "Oso roles" — currently we use "Oso Roles", "Oso roles", and (gasp) "oso roles" to refer to it